platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/generate-platform-diagram.sh

```diff
@@ -12,6 +12,7 @@ pcli diagram \
     -l 'TransactionPrehandler:futures:consensusRoundHandler' \
```

Comment:
I think we need to move that file. I created https://github.com/hashgraph/hedera-services/issues/13263

Reply:
I'm open to the idea of moving this file as a part of a future PR.